devito/ir/equations/equation.py

```diff
@@ -9,7 +9,7 @@
 from devito.symbolics import IntDiv, uxreplace
 from devito.tools import Pickable, Tag, frozendict
 from devito.types import Eq, Inc, ReduceMax, ReduceMin
-from devito.types import MatVecEq, RHSEq
+from devito.petsc.types import MatVecEq, RHSEq
```

Comment:
mmhh this is at risk of circular dependence in the future, but I don't have a better way off top of my head